### PR TITLE
[IMP] website: add new `s_mockup_image` snippet

### PR DESCRIPTION
**Pull request by anso-odoo**
This commit introduces the new `s_mockup_image` content snippet.

Part of task-4077427
task-4094379

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/data/image_library.xml [hunk 1]
```diff
@@ -709,5 +709,11 @@
     <field name="type">url</field>
     <field name="url">/website/static/src/img/snippets_demo/s_popup.jpg</field>
 </record>
+<record id="website.s_mockup_image_default_image" model="ir.attachment">
+    <field name="public" eval="True"/>
+    <field name="name">s_mockup_image_default_image.jpg</field>
+    <field name="type">url</field>
+    <field name="url">/website/static/src/img/snippets_demo/s_mockup_image.jpg</field>
```
Comment (stefanorigano) [inline addons/website/data/image_library.xml:716]: Not a fan of the "website inside a website" effect.. can we use the odoo backend instead?
We discussed about the graphView, but even a nice kanban or the appSwitcher is worth considering IMO 😉

File: addons/website/views/snippets/s_mockup_image.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_mockup_image" name="Mockup Image">
+    <section class="s_mockup_image o_cc o_cc1 pt72 pb72" data-oe-shape-data="{'shape':'web_editor/Wavy/11', 'colors':{'c1': '#2D3142', 'c4': '#FFFFFF'}, 'showOnMobile':true}">
+        <div class="o_we_shape o_web_editor_Wavy_11 o_shape_show_mobile" style="background-image: url(&quot;/web_editor/shape/web_editor%2FWavy%2F11.svg?c1=%232D3142&amp;c4=%23FFFFFF&quot;);"/>
+        <div class="container">
+            <div class="row align-items-center">
+                <div class="col-lg-4 pt16 pb16">
+                    <h2>All your business on one platform.</h2>
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_mockup_image.xml:10]: Remember to adapt the heading text accordingly to the image that you'll choose (the mockup one was not bad IMO)

image

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Thanks @anso-odoo , just 2 minors, well done 👍

File: addons/website/views/snippets/s_mockup_image.xml (hunk 2 above)
Comment (anso-odoo) [reply addons/website/views/snippets/s_mockup_image.xml]: All fixed

**Review by stefanorigano — COMMENTED** [review]
Thanks, same as https://github.com/odoo/odoo/pull/175682#pullrequestreview-2223626618 ;)

File: addons/website/views/snippets/s_mockup_image.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        <div class="container">
+            <div class="row align-items-center">
+                <div class="col-lg-4 pt16 pb16">
+                    <h2>The Innovation Behind Our Product</h2>
+                    <p>Write one or two paragraphs describing your product or services. To be successful your content needs to be useful to your readers.</p>
+                    <p>Start with the customer – find out what they want and give it to them.</p>
+                    <p><a href="#" class="btn btn-primary">Learn more</a></p>
+                </div>
+                <div class="col-lg-8 pt16 pb16">
+                    <img
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_mockup_image.xml:16]: Everything in one line 😉

**Review by stefanorigano — APPROVED** [review]
LGTM 🙌

It's just a pity that the image's text looks pixelated.
Blurred would have been fine/predictable. This pixelated effect it's  a bit unfortunate instead :/
But it's not related to the image I guess, we can move forward 👍 

image

**Comment by anso-odoo** [comment]
@stefanorigano looks like it screen related. I don't have the issue on a retina screen but it's true that it's a bit pixelated on non retina screens

**Comment by bso-odoo** [comment]
@robodoo r+

**Pull request closed without merging**